runtime/battery-station/src/lib.rs

```diff
@@ -112,12 +110,9 @@ impl Contains<Call> for IsCallable {
                 match inner_call {
                     // Disable Rikiddo markets
                     create_market { scoring_rule: RikiddoSigmoidFeeMarketEma, .. } => false,
-                    // Disable Court & SimpleDisputes dispute resolution mechanism
-                    create_market { dispute_mechanism: Court | SimpleDisputes, .. } => false,
-                    create_cpmm_market_and_deploy_assets {
-                        dispute_mechanism: Court | SimpleDisputes,
-                        ..
-                    } => false,
+                    // Disable Court dispute resolution mechanism
+                    create_market { dispute_mechanism: Court, .. } => false,
+                    create_cpmm_market_and_deploy_assets { dispute_mechanism: Court, .. } => false,
```

Comment:
I used `MaxDisputes = 3` for battery-station now. So we have two conflicting MDMs, which with the current setting run with `global-disputes`. It is `authorized` and `simple-disputes`. But `authorized` with `global-disputes` would mean the following. Before 3 disputes the authority makes the final decision. With the third dispute, the authority has no control over the final decision, it's up to the token holders. Does this make any sense or should we disallow `authorized` for the battery station with this setting?

Reply:
A better possibility might be to only allow `start_global_dispute` with the `simple-disputes` MDM.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/682/commits/917f24f2e87496c7e207c2e0ef91c415735c02cf